client/actions/index.js

```diff
@@ -1,36 +1,41 @@
-import { getFruits } from '../apis/fruits'
-import { getVegetables } from '../apis/vegetables'
+import { getRecipes } from '../apis/recipes'
+import { getProduce } from '../apis/produce'
 
-export const SET_FRUITS = 'SET_FRUITS'
+export const DISPLAY_RECIPES = 'DISPLAY_RECIPES'
 
-export const SET_VEGETABLES = 'SET_VEGETABLES'
-
-export function setFruits(fruits) {
+export function displayRecipes(recipes) {
   return {
-    type: SET_FRUITS,
-    payload: fruits,
+    type: DISPLAY_RECIPES,
+    payload: recipes,
   }
 }
 
-export function setVegetables(vegetables) {
-  return {
-    type: SET_VEGETABLES,
-    payload: vegetables,
+export function fetchRecipes(userInput) {
+  return async (dispatch) => {
+    try {
+      const recipes = await getRecipes(userInput)
```

Comment:
Mentioned this in the server folder too but I would recommend added a limit to this api call.
```suggestion
      const recipes = await getRecipes(userInput, 10)
```
ie. this will only return the first 10 recipes